src/schema/value.rs

```diff
@@ -49,6 +49,13 @@ pub enum OwnedValue {
     IpAddr(Ipv6Addr),
 }
 
+impl AsRef<OwnedValue> for OwnedValue {
+    #[inline]
+    fn as_ref(&self) -> &OwnedValue {
+        self
+    }
+}
+
```

Comment:
Yes, just ergonomics, because the Trait is implemented on the reference and the compiler does not auto infer a value to it's reference to access the trait

Reply:
Fair enough